katportalclient/test/test_client.py

```diff
@@ -1062,14 +1062,12 @@ def test_sensor_values_multiple_filters_multiple_matches(self):
 
     @gen_test
     def test_sensor_history_single_sensor_without_value_time(self):
-        """Test that time ordered data with value_time is received for a
+        """Test that time ordered data without value_time is received for a
         single sensor request.
         """
         history_base_url = self._portal_client.sitemap[
             'historic_sensor_values']
         sensor_name = 'anc_mean_wind_speed'
-        publish_messages = [sensor_history_pub_messages_json['init']]
```

Comment:
The definition of `sensor_history_pub_messages_json`  on line 222 still needs to be removed.